helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml

```diff
@@ -97,14 +95,12 @@ spec:
           value: '{{ .Values.engine.grpc.port }}'
         - name: EXECUTOR_SERVER_PORT
           value: '{{ .Values.executor.port }}'
-        - name: EXECUTOR_SERVER_METRICS_PORT_NAME
```

Comment:
Why have the metrics port names disappeared?

Reply:
Probably I need to rebase.

Reply:
Turns out this is from running `make create` in `operator/helm`. Something is out of step there in master.

Reply:
It's because https://github.com/SeldonIO/seldon-core/commit/e86259253a47df08a9d46a3a76ab9ed677b3af4f#diff-611879a2bc550a087e6a3e566a2fa924 didn't update split_resources.py

Reply:
ahh.. maybe worth fixing here?

Reply:
Now incorporated fix here - https://github.com/SeldonIO/seldon-core/pull/1861/files#diff-1fa53ebe62ea3326220b4e91bc47d845R90